app/pb_kits/playbook/pb_typeahead/docs/_typeahead_default.html.erb

```diff
@@ -19,7 +19,8 @@
     name: tag(:slot, name: "name"),
     orientation: "horizontal",
     align: "left",
-    avatar_url: "placeholder"
+    avatar_url: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR4nGP6zwAAAgcBApocMXEAAAAASUVORK5CYII=",
```

Comment:
Please don't use "url" values like the original one. This one at least is a data url. 🤦